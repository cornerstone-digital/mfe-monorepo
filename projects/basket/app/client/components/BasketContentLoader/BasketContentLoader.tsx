import { Grid, GridColumn, GridRow } from '@vfuk/core-grid'
import ContentLoader from '@web-core/components/atoms/ContentLoader'

const BasketContentLoader = () => {
  return (
    <Grid>
      <GridRow noGutters>
        <ContentLoader />
      </GridRow>
      <GridRow noGutters marginBottom={2}>
        <GridColumn col={5} marginBottom={1} marginTop={1}>
          <ContentLoader height="inherit" />
        </GridColumn>
        <GridColumn col={2} offset={2} marginBottom={1} marginTop={1}>
          <ContentLoader />
        </GridColumn>
        <GridColumn col={2} offset={1} marginBottom={1} marginTop={1}>
          <ContentLoader />
        </GridColumn>
      </GridRow>
      <GridRow noGutters>
        <GridColumn col={5}>
          <ContentLoader />
        </GridColumn>
        <GridColumn col={5} offset={2}>
          <ContentLoader />
        </GridColumn>
      </GridRow>
      <GridRow noGutters>
        <GridColumn col={4}>
          <ContentLoader />
        </GridColumn>
        <GridColumn col={2} offset={6}>
          <ContentLoader />
        </GridColumn>
      </GridRow>
      <GridRow noGutters marginTop={2} marginBottom={2}>
        <ContentLoader height="35px" heightLg="50px" />
      </GridRow>
      <GridRow align="center">
        <GridColumn alignSelf="center">
          <ContentLoader height="80px" />
        </GridColumn>
        <GridColumn>
          <ContentLoader height="80px" />
        </GridColumn>
        <GridColumn alignSelf="center">
          <ContentLoader gutters="vertical" height="80px" />
        </GridColumn>
      </GridRow>
    </Grid>
  )
}

export default BasketContentLoader
