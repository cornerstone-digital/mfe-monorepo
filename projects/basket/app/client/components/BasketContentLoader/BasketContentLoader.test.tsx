import enzyme from 'enzyme'
import toJson from 'enzyme-to-json'

import BasketContentLoader from './BasketContentLoader'

test('it should render', () => {
  const wrapper = enzyme.shallow(<BasketContentLoader />)
  expect(wrapper.length).toEqual(1)
  expect(toJson(wrapper)).toMatchSnapshot()
})
