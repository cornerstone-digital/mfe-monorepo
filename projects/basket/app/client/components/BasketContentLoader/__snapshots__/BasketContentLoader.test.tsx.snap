// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it should render 1`] = `
<Grid
  center={true}
  maxWidth="full"
>
  <Grid
    disableFlex={false}
    noGutters={true}
  >
    <ContentLoader
      height="25px"
      width="100%"
    />
  </Grid>
  <Grid
    disableFlex={false}
    marginBottom={2}
    noGutters={true}
  >
    <Grid
      col={5}
      marginBottom={1}
      marginTop={1}
    >
      <ContentLoader
        height="inherit"
        width="100%"
      />
    </Grid>
    <Grid
      col={2}
      marginBottom={1}
      marginTop={1}
      offset={2}
    >
      <ContentLoader
        height="25px"
        width="100%"
      />
    </Grid>
    <Grid
      col={2}
      marginBottom={1}
      marginTop={1}
      offset={1}
    >
      <ContentLoader
        height="25px"
        width="100%"
      />
    </Grid>
  </Grid>
  <Grid
    disableFlex={false}
    noGutters={true}
  >
    <Grid
      col={5}
    >
      <ContentLoader
        height="25px"
        width="100%"
      />
    </Grid>
    <Grid
      col={5}
      offset={2}
    >
      <ContentLoader
        height="25px"
        width="100%"
      />
    </Grid>
  </Grid>
  <Grid
    disableFlex={false}
    noGutters={true}
  >
    <Grid
      col={4}
    >
      <ContentLoader
        height="25px"
        width="100%"
      />
    </Grid>
    <Grid
      col={2}
      offset={6}
    >
      <ContentLoader
        height="25px"
        width="100%"
      />
    </Grid>
  </Grid>
  <Grid
    disableFlex={false}
    marginBottom={2}
    marginTop={2}
    noGutters={true}
  >
    <ContentLoader
      height="35px"
      heightLg="50px"
      width="100%"
    />
  </Grid>
  <Grid
    align="center"
    disableFlex={false}
    noGutters={false}
  >
    <Grid
      alignSelf="center"
    >
      <ContentLoader
        height="80px"
        width="100%"
      />
    </Grid>
    <Grid>
      <ContentLoader
        height="80px"
        width="100%"
      />
    </Grid>
    <Grid
      alignSelf="center"
    >
      <ContentLoader
        gutters="vertical"
        height="80px"
        width="100%"
      />
    </Grid>
  </Grid>
</Grid>
`;
